CharacterCreation/Class-Specific Documentation/Engineer Processes.txt

```diff
@@ -520,6 +520,9 @@ saying this is that if a Drone has 3 IO Ports, you can't build 4 Peripherals
 onto it and have the Drone turn peripherals off or on as it wants to; every
 Peripheral must have an active IO Port or it simply falls off of the Drone. 
 
+    IO Ports are a real-world Computer/Electric Engineering term which is short
+for Input/Output Ports.
```

Comment:
We're still going to lose people over Ports. It's not necessarily a problem, but I see the possibility for supplying learning and interest by adding something like " ... Input/Output Ports: physical connection points that allow two-way data transfer."